src/agent/src/main.rs

```diff
@@ -372,6 +390,10 @@ async fn start_sandbox(
     let (tx, rx) = tokio::sync::oneshot::channel();
     sandbox.lock().await.sender = Some(tx);
 
+    if Path::new(CDH_PATH).exists() && Path::new(AA_PATH).exists() {
+        init_attestation_components(logger, config)?;
+    }
```

Comment:
I think it could be handy to have some debug messages here saying whether or not the attestation components are being started.